docs/ReleaseNotes.md

```diff
@@ -1,6 +1,42 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.0
+Acuminator 2.0 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Key Changes
+Acuminator 2.0 includes the following key changes:
+
+ - Acuminator no longer supports Visual Studio 2015. 
+ - Acuminator now supports C# 7.0 and 8.0.
+
+### Enhancements to Suppression Mechanism
+With the new version of Acuminator, you can suppress a particular diagnostic in a specific place in your project as follows:
+
+ 1. Click the code that is highlighted by the Acuminator diagnostic so that the editor cursor is in the highlighted element.
+ 2. Click **Suppress Acuminator Diagnostic > in Acuminator Suppression File** on the **Acuminator** main menu of Visual Studio.
+
+The suppressed diagnostic is saved in the Acuminator suppression file, which is located in the folder of your Visual Studio project. This file has the same name as the project file and the `acuminator` extension. If there is no Acuminator suppression file in the project folder, the file is created automatically.
+
+Acuminator suppression mechanism differs from the standard suppression mechanism of Visual Studio. With the Acuminator suppression mechanism, you can suppress a particular diagnostic in a specific place in the project, while the standard mechanism suppresses all diagnostics with this ID in the method.
+
+To stop suppression of the diagnostic in the particular place in the code, you remove the diagnostic from the Acuminator suppression file manually.
+
+### Fixed Bugs
+In this version of Acuminator, the following bugs have been fixed:
+
+ - If the Acuminator suppression file had an incorrect XML format, syntax highlighting and other functions stopped working.
+ - The [PX1015](diagnostics/PX1015.md) diagnostic was displayed for `PXSelectExtension.Select` methods. Now the diagnostic is not performed on these methods.
+ - The [PX1048](diagnostics/PX1048.md) diagnostic was displayed if the code contained the `is` pattern. In Acuminator 2.0, the version of Roslyn that is used internally has been updated to 2.0, which supports the `is` pattern.
+ - The analyzer for the [PX1030](diagnostics/PX1030.md) diagnostic failed with an error on some solutions.
+
+### Other Enhancements
+Acuminator 2.0 also includes the following enhancements:
+
+ - The code map stays open when the Visual Studio solution is closed.
+ - Acuminator now stops analysis while the code file is being edited. Acuminator restarts the analysis once the editing stops.
```

Comment:
All analyzers and colorizer too cancel their execution in case of changes done to the file. Still we had this feature since the earliest versions. I just closed the item which was created to add this cancellation to some of the oldest diagnostics. This was done also in the previous versions, just the item remained opened. I don't think it should be mentioned.

Reply:
fixed